src/imp/security_framework.rs

```diff
@@ -225,6 +225,10 @@ impl Certificate {
     pub fn to_der(&self) -> Result<Vec<u8>, Error> {
         Ok(self.0.to_der())
     }
+
+    pub fn to_pem(&self) -> Result<Vec<u8>, Error> {
+        panic!("Not implemented on macOS/iOS");
```

Comment:
This'll need to be implemented.

Reply:
Seems that there's currently no implementation in the underlying [rust-security-framework](https://github.com/kornelski/rust-security-framework) crate. I'll see if I can do anything there.